cmd/proxy-server/main.go

```diff
@@ -738,15 +740,14 @@ func volumesHandler(roleServ *roleClientService, storageServ *storageClientServi
 			}
 
 			storageResp, err = storageServ.storageClient.GetPowerflexVolumes(r.Context(), powerflexVolumesRequest)
-
-			volumeList = append(volumeList, storageResp.Volume...)
-
 			if err != nil {
 				w.WriteHeader(http.StatusInternalServerError)
 				log.WithError(err).Println("unable to get powerflex volumes")
 				return
```

Comment:
If there is an error, the client will only get the status code and no error message. We should write an error similar to how we do in other places in this handler with `web.JSONErrorResponse`.